libs/coin-framework/src/api/types.ts

```diff
@@ -31,6 +31,14 @@ export type Asset = {
   native: bigint;
 };
 
+export type TransactionIntent = {
+  sender: string;
+  recipient: string;
+  amount: bigint;
+  contractAddress?: string;
+  standard?: "trc10" | "trc20";
```

Comment:
The type cannot be tight to Tron implementation.
The easiest way is to use `string` (not sure that Generic will be efficient here)